app/src/main/java/com/example/keniphhomemaintenance/maintenance_screen/CloseMaintenanceDialogButton.kt

```diff
@@ -0,0 +1,28 @@
+package com.example.keniphhomemaintenance.maintenance_screen
+
+import androidx.compose.material.ExtendedFloatingActionButton
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.MutableState
+import androidx.compose.ui.res.stringResource
+import androidx.lifecycle.viewmodel.compose.viewModel
+
+@Composable
+fun CloseMaintenanceDialog(isDialogOpen: MutableState<Boolean>, itemTitle: MutableState<String>, dueDate: MutableState<String>, location: MutableState<String>, stringResource: Int, addMaintenanceItem: Boolean, maintenanceViewModel: MaintenanceViewModel = viewModel()) {
+    ExtendedFloatingActionButton(
```

Comment:
To clarify, I think it's fine to leave the EFAB where it was prior to this commit, and abstract out only the shared _functionality_ of the two. The shared functionality being what happens when the dialog is closed. 

You're going to to get rid of this file, so it doesn't matter a whole lot, but the name of the composable isn't self-documenting. CloseMaintenanceDialog sounds like an action, but this composable provides an EFAB, which isn't obvious by reading the name of the function.